extension/e2e-tests/addAsset.test.ts

```diff
@@ -34,13 +64,12 @@ test.skip("Adding unverified Soroban token", async ({ page, extensionId }) => {
   await expect(page.getByTestId("account-view")).toContainText("E2E");
 });
 test("Adding Soroban verified token", async ({ page, extensionId }) => {
-  const assetsList = await fetch(
-    "https://api.stellar.expert/explorer/testnet/asset-list/top50",
-  );
-  const assetsListData = await assetsList.json();
-  const verifiedToken =
-    assetsListData?.assets[0]?.contract ||
-    "CBIELTK6YBZJU5UP2WWQEUCYKLPU6AUNZ2BQ4WWFEIE3USCIHMXQDAMA";
```

Comment:
I'm removing some unnecessary data fetching here - it's more reliable to just hardcode the USDC token address